src/Request/RequestHandler.php

```diff
@@ -157,6 +168,11 @@ public function __construct(array $config)
             $this->privateKey = $config['private_key'];
         }
 
+        $this->tokenVerified = false;
+        if (isset($config['token_verified'])) {
```

Comment:
3-ledged oauth
1. get request token
2. send it to user to confirm request token
3. get access token
4. on getting access token we set tokenVerified = true
@darrynten